WcaOnRails/lib/qualification.rb

```diff
@@ -35,11 +35,36 @@ def self.load(json)
       rescue ArgumentError
         nil
       end
-      p out
       out
     end
   end
 
+  def can_register?(user, event_id)
+    if user.person.nil?
+      return false
+    end
```

Comment:
Ruby syntactic sugar: `return false if user.person.nil?`

Reply:
(Yes, as a one-liner.)

Reply:
Thank you!